src/test/java/com/kainos/ea/backend/BackendApplicationTests.java

```diff
@@ -3,17 +3,11 @@
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.boot.web.server.LocalServerPort;
-import org.springframework.test.context.TestPropertySource;
 
 @SpringBootTest
-//@TestPropertySource(locations = "/application-test.properties")
 public class BackendApplicationTests {
 
-	@LocalServerPort
-	protected static final int PORT = 8080;
-
-
-	protected static String createURLWithPort(String uri) {
-		return "http://localhost:" + PORT + uri;
+	@Test
+	void contextLoads() {
 	}
```

Comment:
This will crash Travis for now, i suggest commenting it out or removing

Reply:
I saw that and I believe it was because of some inconsistency with band.bandName vs band.name, I changed it in the last commit so should be fixed now. Either way, it's been merged :/